controllers/student.js

```diff
@@ -1,7 +1,35 @@
-const getStudents = async () => {}
-const getStudentById = async () => {}
-const updateStudent = async () => {}
-const deleteStudent = async () => {}
+const studentService = require('~/services/student')
+
+const getStudents = async (_req, res) => {
+  const students = await studentService.getStudents()
+
+  res.status(200).json(students)
+}
+
+const getStudentById = async (req, res) => {
+  const { id } = req.params
+
+  const student = await studentService.getStudentById(id)
+
+  res.status(200).json(student)
+}
+
+const updateStudent = async (req, res) => {
+  const { id } = req.params
+  const updateData = req.body
+
+  const student = await studentService.updateStudent(id, updateData)
+
+  res.status(200).json(student)
```

Comment:
should we actually return here anything? mby 204?

Reply:
fixed